demo/server.js

```diff
@@ -0,0 +1,30 @@
+var express = require('express')
+var bodyParser = require('body-parser')
+var serveStatic = require('serve-static')
+var serveIndex = require('serve-index')
+
+var app = express()
+app.use(bodyParser.urlencoded({ extended: false }))
+
+app.post('/post', function(request, response) {
```

Comment:
It's a DEMO script for development, not for production. Noone should care if it's a function or arrow function, because there is no `this` involved. This comment already cost more bytes and brain power than it would remove.

Reply:
I care, thus I unresolved the conversations that you simply resolved.
If we have a common coding style, we should try to always follow it.